iet.so/include/iet_worker.h

```diff
@@ -208,14 +213,15 @@ class IETWorker : public rvs::ThreadBase {
     void log_to_json(const std::string &key, const std::string &value,
                         int log_level);
 
-
  protected:
     std::unique_ptr<rvs_blas> gpu_blas;
 
     //! name of the action
     std::string action_name;
     //! index of the GPU (as reported by HIP API) that will run the EDPp test
     int gpu_device_index;
+    //! index of GPU (in view of smi lib) which is sometimes different to above index
+    int smi_device_index;
```

Comment:
Same comment as before

Reply:
agreed, but kept naming convention consistent with the existing fields